website/static/js/contribManager.js

```diff
@@ -188,44 +214,49 @@ var ContributorsViewModel = function(contributors, adminContributors, user, isRe
 
     self.original = ko.observableArray(contributors);
 
+    self.permissionDict = {
+        read: 'Read',
+        write: 'Read + Write',
+        admin: 'Administrator'
+    };
+
+    self.permissionList = Object.keys(self.permissionDict);
+
     self.contributors = ko.observableArray();
+
     self.adminContributors = adminContributors;
 
     self.user = ko.observable(user);
-    // TODO: Does this need to be an observable?
-    self.userIsAdmin  = ko.observable($.inArray('admin', user.permissions) !== -1);
     self.canEdit = ko.computed(function() {
-        return (self.userIsAdmin()) && !isRegistration;
+        return ($.inArray('admin', user.permissions) > -1) && !isRegistration;
     });
 
-    self.messages = ko.observableArray([]);
+    self.sortable = function(filtered) {
+        if (!isRegistration && user.is_admin) {
+            if (filtered) {
+                $('#contributors').sortable('disable');
+                self.isSortable(false);
+            }
+            else {
+                $('#contributors').sortable('enable');
```

Comment:
Would it make sense to add a subscriber function to `isSortable`, so that the $(..).sortable() is called whenever isSortable is changed?